### PR TITLE
Adding ISA to the list

### PR DESCRIPTION
**Pull request by crteezy**
Hi @dilumn 
Adding my company and AED currency to the data migration.
Kindly let me know if you need anything else

### CODE REVIEW

File: db/data_migrate/20220414233400_add_isa_to_list.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class AddCpToList < ActiveRecord::Migration[7.0]
+  def change
+    Organization.create!(
+      name: "ISA / Accelaero",
+      pegged: true,
+      great_place_to_work: false,
+      pegged_currency: 'AED',
+      remarks: 'Salary will be fully pegged. EPF, ETF will be calculated on the pegged salary. Will be applicable from April',
+      pegging_value: '55 AED',
```
Comment (dilumn) [inline db/data_migrate/20220414233400_add_isa_to_list.rb:9]: `pegging_value` is an integer https://github.com/dilumn/peggedplacetowork/blob/master/db/schema.rb#L43, so this need to be `pegging_value: 55,`

**Review by dilumn — CHANGES_REQUESTED** [review]
@crteezy thank you for the pull request. One small change & we are good to go. I will merge once that change is done

File: db/data_migrate/20220414233400_add_isa_to_list.rb (hunk 1 above)
Comment (crteezy) [reply db/data_migrate/20220414233400_add_isa_to_list.rb]: PR updated. thanks

**Review by dilumn — APPROVED** [review]

**Comment by dilumn** [comment]
Data migrated to production

**Pull request merged**